cob_bringup/components/legacy_schunk_powercube.launch

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0"?>
+<launch>
+	<arg name="robot"/>
+	<arg name="component_name"/>
+	<arg name="sim" default="false"/>
+
+	<group unless="$(arg sim)">
+		<include file="$(find cob_bringup)/drivers/schunk_powercube_chain_driver.launch" >
+		  <arg name="robot" value="$(arg robot)" />
+		  <arg name="component_name" value="$(arg component_name)" />
+		</include>
+		<include file="$(find cob_bringup)/controllers/cob_trajectory_controller.launch">
```

Comment:
imo, it would improve readability to add an empty line between includes. what do you think?